content/03.methods.md

```diff
@@ -0,0 +1,36 @@
+## Methods
+
+### Mutation data download and preprocessing
+
+To generate binary mutated/non-mutated gene labels for our machine learning model, we used mutation calls for TCGA samples from MC3 [@doi:10.1016/j.cels.2018.03.002] and copy number threshold calls from GISTIC2.0 [@doi:10.1186/gb-2011-12-4-r41].
+MC3 mutation calls were downloaded from the Genome Data Commons (GDC) of the National Cancer Institute, at [`https://gdc.cancer.gov/about-data/publications/pancanatlas`](https://gdc.cancer.gov/about-data/publications/pancanatlas).
+Copy number threshold calls are from an older version of PanCanAtlas, and are available here: [`https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122`](https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122).
+We removed hypermutated samples (defined as five or more standard deviations above the mean non-silent somatic mutation count) from our dataset, to avoid potential false positives.
+In total, this resulted in 9,074 TCGA samples with mutation and copy number data that were used in our analyses.
+Any sample with a non-silent somatic variant in the target gene was included in the positive set.
+We also included copy number gains in the target gene for oncogenes, and copy number losses in the target gene for tumor suppressor genes, in the positive set; all remaining samples were considered negative for mutation in the target gene.
+
+### Omics data download and preprocessing
+
+RNA sequencing, 27K and 450K methylation array, and RPPA datasets for TCGA samples were all downloaded from GDC, at the same link provided above.
+Mutational signatures information for TCGA samples with whole-exome sequencing data was downloaded from the International Cancer Genome Consortium (ICGC) data portal, at [`https://dcc.icgc.org/releases/PCAWG/mutational_signatures/Signatures_in_Samples/SP_Signatures_in_Samples`](https://dcc.icgc.org/releases/PCAWG/mutational_signatures/Signatures_in_Samples/SP_Signatures_in_Samples).
+For our experiments, we used only the "single base signature" (SBS) mutational signatures, generated in [@doi:10.1038/s41586-020-1943-3].
+We standardized (took z-scores of) each column of RNA sequencing and RPPA data; methylation data and mutational signatures data were left untransformed (beta values and mutation counts respectively), except in multi-omics experiments where all data types were standardized.
+For the RNA sequencing dataset, we used only the top 8,000 gene features by mean absolute deviation as predictors in our models, except in multi-omics experiments where all 15,639 genes were used.
```

Comment:
Just to check - if you use all gene features does performance look different than 8k? Trying to think of why expression might be "good". I'm guessing not because of the multi-omic statement later but 🤷‍♂️

Reply:
I vaguely remember using all gene features a while ago and seeing that performance was essentially the same, but maybe I'll run it again this weekend with the current version of the pipeline, just to make sure.

It probably is a good thing to check, I agree.